.github/pull_request_template.md

```diff
@@ -15,6 +15,7 @@
 - [ ] Code refactor (non-breaking change that doesn't add functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
+- [ ] Configuration change
```

Comment:
The goal of this list is for us to get a glimpse into what to write in our Release Notes.
Ultimately more important than the labels we attributed to each one of these points is the description, so maybe we should change that instead. It might be best renaming `Code refactor` to `Non-breaking change (non-breaking change that doesn't add functionality - ie. code refactor, configs, etc.)`

My reasoning is that a configuration change that changes no behavior for users of the gem should be a `non-breaking change that doesn't add functionality`. It is possible that a config change does break functionality for users, and so it should go into the `breaking change` category.

Wdyt?

Reply:
I think your reasoning makes sense here. I wonder if it makes sense to reword the list a bit if we use it to help us write our release notes 🤔 What do you think of this:

```markdown
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-breaking change (a change that doesn't alter functionality - i.e., code refactor, configs, etc.)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] It is safe to rollback this change.
```